packages/generator/templates/app/README.md

```diff
@@ -91,12 +118,16 @@ These files are:
 
 - `public/` is a directory where you will put any static assets. If you have images, files, or videos which you want to use in your app, this is where to put them.
 
+- `test/` is a directory where you can put your automated tests.
+
 - `utils/` is a good place to put any shared utility files which you might use across different sections of your app.
 
 - `.babelrc.js`, `.env`, etc. ("dotfiles") are configuration files for various bits of JavaScript tooling.
 
 - `blitz.config.js` is for advanced custom configuration of Blitz. It extends [`next.config.js`](https://nextjs.org/docs/api-reference/next.config.js/introduction).
 
+- `jest.config.js` is a file where you can find Jest's configuration.
```

Comment:
We should link out to the Jest docs here with a note that it can be safely customized